app/src/main/java/com/hedvig/app/feature/offer/OfferViewModel.kt

```diff
@@ -150,4 +178,11 @@ class OfferViewModelImpl(
             response.getOrNull()?.data?.let { offerRepository.removeStartDateFromCache(it) }
         }
     }
+
+    override fun trackUserSign() {
+        viewModelScope.launch {
+            val price = offerRepository.offer().firstOrNull()?.priceOrNull() ?: 0.0
```

Comment:
I don't think that `0.0` is an appropriate default here - perhaps we could do something like this, instead?

```suggestion
            val price = offerRepository.offer().firstOrNull()?.priceOrNull() ?: return@launch
```